vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -15,7 +15,7 @@ $ErrorActionPreference = "Stop"
 
 filter Timestamp { "$(Get-Date -Format o): $_" }
 
-$global:containerdPackageUrl = "https://acs-mirror.azureedge.net/containerd/windows/v0.0.1/binaries/containerd-v0.0.1-windows-amd64.tar.gz"
+$global:containerdPackageUrl = "https://k8sreleases.blob.core.windows.net/containerd/windows/v0.0.41/binaries/containerd-v0.0.41-windows-amd64.tar.gz"
```

Comment:
Can it be downloaded by ff/mc? Or do we only public preview in global azure? If cx has firewall, it may not be downloaded

Reply:
Updated to use acs-mirrored one. Thanks